### DISCUSSION TITLE
Ts-253d

### DISCUSSION THREAD

**Discussion by nzapodege** [pdulvp/jellyfin-qnap#15] [General]
Hi, I tried the beta jellyfin_10.8.0-b2.qpkg and the jellyfin_10.8.0-b3.qpkg, both of them full cpu with low quality movies, hd and 4k movies work fine. beta b1 works fine. I use Intel encoding and I have a "Qnap Ts253-d" with 8GB of ram. What can I do to fix the problem?

**Comment by Sozzi1968** [comment]
I installed the new release 10.9.7. 10.9.7 appears on the server. Web page shows the old 10.9.6 . (Qnap Ts-253d)

**Reply by Sozzi1968** [reply]
I don't know how but after two hours everything is ok. Solved by itself

**Comment by DreamT80** [comment]
Hi,
Late to the party here...But I have a question: What settings are you using for HW transcoding? Intel QSV I suppose, but the other settings? I'm using "/dev/dri/renderD128", but I don't know if this is correct.
The logs say that it's transcoding, but the CPU is very high and the performance isn't the best..

**Reply by Goatrider1** [reply]
Hi, 
the ts-253d is not the most powerful NAS. After the initial jellyfin setup just scanning the library and so on will cause very high CPU for quite some time. It can work properly on a ts-253d, but it's also depending on your environment (movies quality + codecs, concurrent users, other apps, ram etc.). 
Intel QSV with "/dev/dri/renderD128" is correct. If you want to double check, connect via SSH and type "ls -al /dev/dri/renderD128".
Cheers

**Reply by Sozzi1968** [reply]
I increased the RAM from 4GB to 16GB and now it works much better
